flatdata-py/pyproject.toml

```diff
@@ -7,7 +7,6 @@ name = "flatdata-py"
 version = "0.4.7"
 description = "Python 3 implementation of Flatdata"
 readme = "README.md"
-license = ""
```

Comment:
TIL: https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#classifiers

Reply:
Yes, we already have one a few lines below